.tekton/collector-build.yaml

```
@@ -47,6 +47,10 @@ spec:
     value: 'true'
   - name: build-source-image
     value: 'true'
+  - name: extra-labels
+    value:
+    # X.Y in the cpe label must be adjusted for every version stream.
+    - "cpe=cpe:/a:redhat:advanced_cluster_security:X.Y::el8"
```

```suggestion
    - "cpe=cpe:/a:redhat:advanced_cluster_security:4.9::el8"
```

Without this the change won't solve it.